spec/lib/description_remover_spec.rb

```diff
@@ -0,0 +1,102 @@
+require 'rails_helper'
+require 'description_remover'
+require 'gds_api/test_helpers/content_store'
+
+include ::GdsApi::TestHelpers::ContentStore
+include ::GdsApi::TestHelpers::PublishingApiV2
+
+RSpec.describe DescriptionRemover do
+  context "valid taxons are updated" do
```

Comment:
I'm not sure what "valid" means here, do you mean "appropriate"?

Reply:
👍